File: src/kyu_7/building_blocks/python/building_blocks.py
class Block:
    def __init__(self, data):
        self.width = data[0]
        self.length = data[1]
        self.height = data[2]
        
        
    def get_width(self):
        return self.width
    
    def get_length(self):
        return self.length
    
    def get_height(self):
        return self.height
    
    def get_volume(self):
        return self.width * self.length * self.height
    
    def get_surface_area(self):
        return 2 * (self.length * self.width + self.length * self.height + self.width * self.height)
    

File: src/kyu_7/building_blocks/csharp/building_blocks.cs
public class Block
{
    private int width;
    private int length;
    private int height;

    public Block(int[] data)
     {
        width = data[0];
        length = data[1];
        height = data[2];
    }

    public int GetWidth()
    {
        return width;
    }

    public int GetLength()
    {
        return length;
    }

    public int GetHeight()
    {
        return height;
    }

    public int GetVolume()
    {
        return width * length * height;
    }

    public int GetSurfaceArea()
    {
        return 2 * length * width + 2 * length * height + 2 * width * height;
    }
}